configuration/index.md

```diff
@@ -74,58 +74,18 @@ Now that your Companion has an internet connection, you can perform the update t
    ![Version Chooser](version-chooser.png)
 1. If you're already on the latest version, the right side of your Local Version will be blank. If not, you should see a blue **Update** button.
 
-   ![Version Update](version-update.png)
-1. It's also possible to update between different versions (in both directions)
+   <img src="version-update.png" width="650">
+2. It's also possible to update between different versions (When pirate mode is enabled)
```

Comment:
I think repeated `1.` is actually better for this kind of thing - it gets automatically rendered into correct numbering, and makes it possible to switch steps around or add/remove them without needing to re-number everything else.

Reply:
That probably was my code editor, sorry for that.